### PR TITLE
Slight Buff to Genetics

### PR DESCRIPTION
**Pull request by Aurorablade**
NO ONE THOUGHT I WAS INSANE ENOUGH TO DO THIS

:cl:Fethas
rscadd: *fart now has newtonian move if you have the Superfart power
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -819,6 +819,9 @@
 				src.gib()
 			else if(TOXIC_FARTS in mutations)
 				message = "<b>[src]</b> unleashes a [pick("horrible","terrible","foul","disgusting","awful")] fart."
+			else if(SUPER_FART in mutations)
+				message = "<b>[src]</b> unleashes a [pick("loud","deafening")] fart."
+				newtonian_move(dir)
```
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/human/emote.dm]: Ahahahahahahhahahahaha. Oh my gosh.

**Comment by Fox-McCloud** [comment]
:+1: x :100:

**Comment by TullyBurnalot** [comment]
FINALLY, MY DREAM CAN COME TRUE!

**Comment by Crazylemon64** [comment]
unironically buffing science \</s>

**Comment by Dumbdumn5** [comment]
I too find it incredibly useful to expel copious amounts of air out of my rectum to move around my zero-g environment.

**Comment by Earthdivine** [comment]
Oh gods....

**Comment by KasparoVy** [comment]
Lmfao

This is going to be bloody hilarious

**Comment by taukausanake** [comment]
I have no idea what `newtonian_move` does. Someone explain? Is it literally moving in gravity stuff?

**Pull request merged**